### PR TITLE
Lev 406

### PR DESCRIPTION
**Pull request by aarongill1**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
+    https://keycloak-ho-hmpo-hybdidm-idm.ebsa.homeoffice.gov.uk
+    
+    Production:
+    
+    https://keycloak-ho-hmpo-prp1-e-hybdidm-idm.np.ebsa.homeoffice.gov.uk
+
   contact:
     email: levapi@digital.homeoffice.gov.uk
-  version: 0.0.1
+  version: 0.22.0
```
Comment (ashley-bickerstaff-homeoffice) [inline openapi.yaml:30]: What's with the large version jump?

File: openapi.yaml [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
   title: Life Event Verification (LEV) API
-  description: "A ReSTful API for verifying 'life events' such as births, deaths and\
-    \ marriages."
+  description: |
+    "A ReSTful API for verifying 'life events' such as births, deaths and marriages."
+    
+    # Environments
+    The Life Event Verification service is deployed in multiple environments , many to facilitate development and testing. 
+    
+    For customer usage,only the pre-production (prp1) and production (prod) environments are used.
```
Comment (ashley-bickerstaff-homeoffice) [inline openapi.yaml:15]: Space required after the comma

File: openapi.yaml [hunk 3]
```diff
@@ -1,18 +1,33 @@
...
+    description: Production
 info:
   title: Life Event Verification (LEV) API
-  description: "A ReSTful API for verifying 'life events' such as births, deaths and\
-    \ marriages."
+  description: |
+    "A ReSTful API for verifying 'life events' such as births, deaths and marriages."
+    
+    # Environments
+    The Life Event Verification service is deployed in multiple environments , many to facilitate development and testing.
```
Comment (ashley-bickerstaff-homeoffice) [inline openapi.yaml:13]: Can you remove the space before the comma?

File: openapi.yaml [hunk 4]
```diff
@@ -1,18 +1,33 @@
...
+    The Life Event Verification service is deployed in multiple environments , many to facilitate development and testing. 
+    
+    For customer usage,only the pre-production (prp1) and production (prod) environments are used.
+    
+    # Keycloak/Bearer Tokens
+    LEV uses Keycloak to issue Bearer Tokens. There are different instances that can be used, depending on environment.
+    
+    Non-production:
+    
+    https://keycloak-ho-hmpo-hybdidm-idm.ebsa.homeoffice.gov.uk
```
Comment (ashley-bickerstaff-homeoffice) [inline openapi.yaml:22]: I know this is 'guessed' at the moment. But it looks like the prod and pre-prod ones are the wrong way around.

**Review by ashley-bickerstaff-homeoffice — CHANGES_REQUESTED** [review]
Few changes requested. Would like to get a quick glance again before merge

File: openapi.yaml (hunk 1 above)
Comment (aarongill1) [reply openapi.yaml:30]: Our previous docs had this in-line with the api and I was under the impression it was the api version and not the spec. 

This turns out to be wrong, so updated to 0.0.2

File: openapi.yaml (hunk 2 above)
Comment (aarongill1) [reply openapi.yaml:15]: Updated

File: openapi.yaml (hunk 3 above)
Comment (aarongill1) [reply openapi.yaml:13]: Done

File: openapi.yaml (hunk 4 above)
Comment (aarongill1) [reply openapi.yaml:22]: Right you are -  swapped around

**Review by ashley-bickerstaff-homeoffice — APPROVED** [review]

**Pull request merged**